src/lib.rs

```diff
@@ -278,6 +280,64 @@ impl From<Ulid> for u128 {
     }
 }
 
+impl From<[u8; 16]> for Ulid {
+    fn from(bytes: [u8; 16]) -> Self {
+        let value = u128::from(bytes[0]) << 120
+            | u128::from(bytes[1]) << 112
+            | u128::from(bytes[2]) << 104
+            | u128::from(bytes[3]) << 96
+            | u128::from(bytes[4]) << 88
+            | u128::from(bytes[5]) << 80
+            | u128::from(bytes[6]) << 72
+            | u128::from(bytes[7]) << 64
+            | u128::from(bytes[8]) << 56
+            | u128::from(bytes[9]) << 48
+            | u128::from(bytes[10]) << 40
+            | u128::from(bytes[11]) << 32
+            | u128::from(bytes[12]) << 24
+            | u128::from(bytes[13]) << 16
+            | u128::from(bytes[14]) << 8
+            | u128::from(bytes[15]);
+
+        Self(value)
+    }
+}
+
+impl From<Ulid> for [u8; 16] {
```

Comment:
Can add a `const fn Ulid::to_bytes(): [u8; 16]` function to allow this to be used in a const context.

Reply:
Done!